guide/07-working-with-bigdata/08-performing_data_enrichment.ipynb

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
<p>The <code><a href="https://developers.arcgis.com/rest/services-reference/enterprise/enrich-from-multi-variable-grid.htm" target="_blank">enrich_from_grid</a></code> tool joins attributes from a multivariable grid to a point layer. The multivariable grid must be created using the <code>build_multivariable_grid</code> task. Metadata from the multivariable grid is used to efficiently enrich the input point features, making it faster than the Join Features task. Attributes in the multivariable grid are joined to the input point features when the features intersect the grid. </p><p>The attributes in the multivariable grid can be used as explanatory variables when modeling spatial relationships with your input point features, and this task allows you to join those attributes to the point features quickly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>